docs/source/slurmguide/slurmarray.md

```diff
@@ -34,34 +30,25 @@ taskId = sys.getenv('SLURM_ARRAY_TASK_ID')
 :::
 ::::
 
-When submitting an array and setting its size with many dimensions, please use the `%` symbol to indicate how many tasks run simultaneously. For example, the following code specifies an array of 600 jobs, with 20 running at a time:
-
-:::{code} bash
-#!/bin/bash
-#SBATCH --partition=short
-#SBATCH --job-name=jarray-example
-#SBATCH --output=out/array_%A_%a.out
-#SBATCH --error=err/array_%A_%a.err
-#SBATCH --array=1-600%20
-:::
-
-Whenever you specify the memory, number of nodes, number of CPUs, or other specifications, they will be applied to each task. Therefore, if we set the header of our submission file as follows:
+## Resource Allotment in Job Arrays 
+When submitting an array and setting its size with many dimensions, please use the `%` symbol to indicate how many tasks run simultaneously. Whenever you specify the memory, number of nodes, number of CPUs, or other specifications, they will be applied to each task. For example, the following code specifies an array of 600 jobs, with 20 running at a time:
 
 :::{code}
 #!/bin/bash
 #SBATCH --partition=short
-#SBATCH --job-name=jarray-example
-#SBATCH --output=out/array_%A_%a.out
-#SBATCH --error=err/array_%A_%a.err
+#SBATCH --job-name=job-array-example
+#SBATCH --output=out-array_%A_%a.out
+#SBATCH --error=err-array_%A_%a.err
 #SBATCH --array=1-600%20
 #SBATCH --mem=128G
 #SBATCH --nodes=2
 :::
+
 Slurm will submit 20 jobs simultaneously. Each job, represented by a task ID, will use two nodes with 128GB of RAM each. In most cases, setting up a single task is sufficient.
 
 Lastly, we usually use job arrays for embarrassingly parallel jobs. If your case is such that the job executed at each job ID does not use any multi-threading libraries, you can use the following header to avoid wasting resources:
```

Comment:
```suggestion
Job arrays are used for embarrassingly parallel jobs. If your job executed at each job ID does not use any multi-threading libraries, you can use the following header to avoid wasting resources:
```